### PR TITLE
Some makefile fixes

### PR DESCRIPTION
**Pull request by bebatut**
- Add make target to help conda env installation 
- Use only gem for installation
- Remove Gemfiles
- Update environment file
- Update package files
- Fix installation command
- Remove bundle usage

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
 _pdf
 .directory
 Gemfile.lock
 build
 node_modules
 miniconda.sh
-decktape-*/
\ No newline at end of file
+decktape-*/
+vendro
```
Comment (shiltemann) [inline .gitignore]: `s/vendro/vendor/` ?

**Comment by jmchilton** [comment]
This should be merged ASAP - it is amazing, I've tried a couple times over the last couple weeks to get my local build going and made no real progress. Worked first try with this branch!

**Comment by bebatut** [comment]
@jmchilton I agree with you: we should merge it soon. I am currently trying to fix the tests on Travis...

**Comment by jmchilton** [comment]
Ahh - didn't notice they were failing - I thought this was just waiting for a merge. Thanks for working on this @bebatut I really appreciate it - I know you are at that conference please don't miss out on that for this.

**Comment by shiltemann** [comment]
thanks a lot @bebatut!

**Comment by martenson** [comment]
xref to heroku trouble thread: https://github.com/galaxyproject/training-material/pull/813

**Pull request merged**